### DISCUSSION TITLE
Actual "user" documentation?

### DISCUSSION THREAD

**Discussion by cdntinker** [Quill-OS/quill#52] [General]
I seem to have successfully installed InkBox onto a beat up Kobo Nia.  This is cool as its original firmware seems to have died.

But...

I have little idea how to actually use it and see no user documentation.

Is anyone working on (or even thinking about) some sort of documentation for end users?

**Comment by Szybet** [comment]
could be made, could be added :)
For now I think you can just click arround things, watch the youtube videos and ask if needed

**Reply by cdntinker** [reply]
Unfortunately, "clicking around things" isn't as intuitive as one might think.
& is no help at all with questions like "How do I manage uploading books via USB?"

Searching for answers seems to be a good way to learn all about temporary tattoos...  :|

**Reply by Szybet** [reply]
1. connect the ereader to the pc
2. click OK on the dialog that appeared
3. upload your books
4. unmount the ereader
5. unplug the cable
6. wait
7. that's it :D

**Reply by cdntinker** [reply]
Worked that out...
Also worked out that I can put the books into folders & have them sorted that way on the device.
But these are the sort of things usually found in documentation.
Pretty sure my 82 yer old Dad would have kept this Kobo in use longer if it hadn't bricked itself.  Which is how I ended up messing around with it.  Definitely can't hand it back as a spare with InkBox on it if he can't learn how to use it that way without digging through the tattooing answers on Google.

**Reply by Szybet** [reply]
This is a new project
It has 2 developers
Please understand this.
We can't do everything, we are looking for contributors, or any kind of support.

**Reply by cdntinker** [reply]
Fully understand.
Sad fact of life, many projects have documentation as the lowest priority part.

**Reply by Szybet** [reply]
Our priority first is to make it usable

**Comment by tux-linux** [comment]
I have asked someone to do it yesterday. Basic end-user documentation
should probably be out at the middle of July.
As @Szybet said, we are not a company, are receiving next to none financial or moral support/contributions from external sources and we do this project when we have time for.

**Reply by Szybet** [reply]
I would add to this "Maybe, most propably"
